apis/filesystem/src/main/java/org/jclouds/filesystem/strategy/internal/FilesystemStorageStrategyImpl.java

```diff
@@ -115,6 +116,7 @@
    private static final String XATTR_USER_METADATA_PREFIX = "user.user-metadata.";
    private static final byte[] DIRECTORY_MD5 =
            Hashing.md5().hashBytes(new byte[0]).asBytes();
+   private static final String MPU_ETAG_FORMAT = "^\"[a-f0-9]{32}-\\d+\"$";
```

Comment:
Prefer `format = Pattern.compile()` and `format.matcher().matches`.  Also are ^$ necessary since you use `match` instead of `find`?

Reply:
Done